lib/experimental/Layouts/Utils/Tag/index.tsx

```diff
@@ -23,21 +25,27 @@ export const Tag = forwardRef<HTMLDivElement, TagProps>(
     return (
       <div
         ref={ref}
+        role={onClick ? "button" : undefined}
         className={cn(
-          "flex flex-row items-center justify-start gap-1.5 rounded-md border border-solid border-f1-border py-1 pl-1 pr-2 font-medium text-f1-foreground",
-          onClick && "cursor-pointer"
+          "group flex w-full flex-row items-center justify-between gap-1.5 rounded-sm py-1.5 pl-1.5 pr-2 font-medium text-f1-foreground",
+          !!onClick && "cursor-pointer hover:bg-f1-background-secondary"
         )}
         onClick={onClick}
       >
-        <span>
+        <span className="flex flex-row items-center gap-1.5">
           <Avatar
             alt={short}
             src={avatar?.src}
             size="xsmall"
             color={getColorFromText(text)}
           />
+          <p>{text}</p>
```

Comment:
it is an invalid html, it is a block element inside an inline (`span`).

Let's use `div` instead of the `span` and this will do the trick